.github/workflows/docker.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   build:
+    if: github.event.pull_request.draft == false
```

Comment:
Will this affect "push" runs as well?

Reply:
it shouldn't conflict bc we have the 
on: 
   push
   
which triggers every time but the condition for enabling the job is always **when not draft** 
so only when you push and PR is not draft CI will run